packages/foundation-models/src/model-types.ts

```diff
@@ -0,0 +1,20 @@
+type LiteralUnion<T extends U, U = string> = T | (U & Record<never, never>);
```

Comment:
[req] I agree with Matthias to not have duplicate code (even the `LiteralUnion` is defined in both packages). I would imagine something like this works:

```
type AzureModels = 'gpt-4o-mini'
  | 'gpt-4o'
  | 'gpt-4'
  | 'gpt-4-32k'
  | 'gpt-35-turbo'
  | 'gpt-35-turbo-16k'
/**
 * Azure OpenAI models for chat completion.
 */
export type AzureOpenAiChatModel = LiteralUnion<AzureModels>;


export type AzureOpenAiChatModelOrchesration = LiteralUnion<Exclude<AzureModels, 'gpt-4o-mini'>>;

```

Reply:
Done.